src/sentry/static/sentry/app/components/userBadge.jsx

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Avatar from './avatar';
+import Link from './link';
+
+const UserBadge = ({user, orgId}) => {
```

Comment:
Is this something we can use for Org members list as well?

Reply:
yeah that's the idea